index.js

```diff
@@ -61,9 +67,24 @@ module.exports = {
             testCredentials = {};
         }
 
+        if (typeof testFlow === "string") {
+                testFlow = JSON.parse(testFlow);
+        }
+
+        var flowId = testFlow[0].z || "f1";
```

Comment:
I like that the tab is included automatically.  I will need to reverse my PR merge to master on my fork.